src/dracula.yml

```diff
@@ -26,21 +26,38 @@ scheme:
   FindMatch: &FindMatch         '#f8f8f220'
   
   # Opaque Colors
-  PURPLE30: &PURPLE30  '#bd93f930'
-  YELLOW50: &YELLOW50  '#f1fa8c50'
+  PURPLE30: &PURPLE30 '#bd93f930'
+  PURPLE90: &PURPLE90 '#bd93f990'
+  PURPLEAA: &PURPLEAA '#bd93f9AA'
+  YELLOW30: &YELLOW30 '#f1fa8c30'
+  YELLOW50: &YELLOW50 '#f1fa8c50'
```

Comment:
I completely agree, in some situations, I would have used the original color if the foreground color weren't forced as white. I can definitely cut down on these if you'd like.